Makefile

```diff
@@ -151,6 +147,31 @@ manifests-generate: $(CONTROLLER_GEN)
 manifests-kustomize: $(KUSTOMIZE)
 	$< build config/default > config/render/capm3.yaml
 
+.PHONY: set-manifest-image-bmo
+set-manifest-image-bmo: manifests $(KUSTOMIZE)
+	$(info Updating container image for BMO to use ${MANIFEST_IMG}:${MANIFEST_TAG})
+	cd config/default && $(KUSTOMIZE) edit set image quay.io/metal3-io/baremetal-operator=${MANIFEST_IMG}:${MANIFEST_TAG}
+
+.PHONY: set-manifest-image-ironic
+set-manifest-image-ironic: manifests $(KUSTOMIZE)
+	$(info Updating container image for Ironic to use ${MANIFEST_IMG}:${MANIFEST_TAG})
+	cd ironic-deployment/default && $(KUSTOMIZE) edit set image quay.io/metal3-io/ironic=${MANIFEST_IMG}:${MANIFEST_TAG}
```

Comment:
In CAPM3 we do a global substitution of all `image: *` in the manifest, but this does not work well with Ironic, since we have multiple different containers (e.g. mariadb). For this reason I decided to use `kustomize edit set image` instead so we replace only the mentioned images.